x/bank/keeper/keeper.go

```diff
@@ -403,6 +403,11 @@ func (k BaseKeeper) SendCoinsFromModuleToModule(
 		return k.addCoins(ctx, recipientAcc.GetAddress(), amt)
 	}
 
+	if amt.IsZero() {
+		return nil
+	}
+
+	k.Logger(ctx).Debug("Sending coins from module to module", "sender", senderModule, "sender_address", senderAddr.String(), "recipient", recipientModule, "recipientAddr", recipientAcc.GetAddress().String(), "amount", amt.String())
```

Comment:
nit, casing for `sender_address` vs `recipientAddr`

Reply:
shot sorry it was on automerger :

Reply:
https://github.com/sei-protocol/sei-cosmos/pull/231